### ISSUE TITLE
Orthogroups division in large dataset

### ISSUE THREAD

**Issue by Matt-BF** [davidemms/OrthoFinder#191]
Good morning,

Our group has used Orthofinder with great sucess to study 18 genomes from species of a taxonomic class. We have since then extended the study to a larger dataset of species from the same class (182 genomes). When comparing the number of Orthogroups, the latter analysis shows a greater number than the former, which suggests a subdivision of some Orthogroups into a greater number (ex: a single functional family on the previous analysis was divided into 50 Orthogroups). Also, there are no Single-Copy Orthogroups in this new analysis.
Is this behaviour expected? Is there a way to minimize this subdivision we are seeing?
Thanks in advance!
Matt

**Comment by davidemms** [comment]
Hi Matt

I'm glad OrthoFinder has been useful. The splintering doesn't sound ideal, have you looked a gene tree containing all the genes that you believe should be in one orthogroup but have been splint into 50? If the genes look like they should be in a single orthogorup then I'd be keen to see it as that isn't something orthofinder should be doing and could be improved.

In terms of the number of orthogroups and lack of single-copy orthogroups I think that is largely expected. If no orthogroups are splintered there will still be orthogroups with genes in just two (or a few) species due to gene birth within the clade and to a lesser extent gene loss. The increase in orthogroup number is probably mostly due to a large number of these small orthogroups of new genes. Similarly, it will be very rare to find a gene which has not be duplicated or lost in a single one of the 182 species hence single copy orthogroups rapidly become rare as species are added. For species tree inference etc. we commonly look for genes that are single copy in a high percentage of species. Hope this is helpful.

All the best
David

**Comment by Matt-BF** [comment]
Hello professor, thanks for the quick reply!

We have the gene tree from the previous analysis and will get back to you in the next few days when I have the results for the new orthogroups.
Also, your explanation for the lack of single-copy orthogroups has been very enlightening, I will keep it in mind when inferring the species' tree.
Thank you once again,
Matt

**Comment by Matt-BF** [comment]
Hello again professor, sorry for the delay
Upon closer inspection of the chosen genomes for our analysis, I have noticed that some of them were low quality, which did some weird things when we ran gene prediction and when naming each gene id. I reran the Orthofinder analysis without these problematic genomes and our problems disappeared, so apparently they were the reason the number of orthogroups was inflated.
The new results actually showed less of some gene families we are interested in when compared to our previous results with 18 species, which is what we expected to see.
Thanks again for your attention, and sorry for the bother,
Matt
